Tasks/AzureFunctionOnKubernetesV0/src/utils/utilities.ts

```diff
@@ -0,0 +1,10 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { KubernetesConnection } from 'kubernetes-common-v2/kubernetesconnection';
+import * as FileHelper from './fileHelper';
+
+export function getKubernetesConnection(): KubernetesConnection {
+    const kubernetesServiceConnection = tl.getInput('kubernetesServiceConnection', true);
+    const tempPath = FileHelper.getTaskTempDir();
+    const connection = new KubernetesConnection(kubernetesServiceConnection, tempPath);
+    return connection;
```

Comment:
NIT: these 3 lines can be converted 1 like:
`return new KubernetesConnection(kubernetesServiceConnection, FileHelper.getTaskTempDir());` instead

Reply:
Done